main.cpp

```diff
@@ -15,7 +15,9 @@ int main(int argc, char* argv[])
   //close the file right away so if the program fails anywhere else, the log file won't be left open.
   //the log file is opened anywhere it needs to be read from or written to, so it is ok to close it here.
   std::ofstream ofile("request_response_log.txt");
```

Comment:
is this necessary? if we open it whever we need to down the read it'll be created on open anyway right?